huaweicloud/services/fgs/resource_huaweicloud_fgs_function.go

```diff
@@ -1598,6 +1615,7 @@ func resourceFunctionRead(_ context.Context, d *schema.ResourceData, meta interf
 			function, make(map[string]interface{})).(map[string]interface{}))),
 		d.Set("gpu_type", utils.PathSearch("gpu_type", function, nil)),
 		d.Set("gpu_memory", utils.PathSearch("gpu_memory", function, nil)),
+		d.Set("lts_custom_tag", utils.PathSearch("lts_custom_tag", function, nil)),
```

Comment:
According to the latest specifications, it is required to add an all map attribute field, and the original map field will not be backfilled.